{"@t":"2024-06-20T04:18:55.1544840Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":127,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-20T04:23:59.2880145Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":22,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-20T04:29:03.3960305Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":168,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-20T04:34:07.5167770Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":209,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
